proto/sge/reward/query.proto

```diff
@@ -21,7 +21,7 @@ service Query {
 
   // Queries a specific Campaign item.
   rpc Campaign(QueryCampaignRequest) returns (QueryCampaignResponse) {
-    option (google.api.http).get = "/sge-network/sge/reward/campaign/{uid}";
+    option (google.api.http).get = "/sge-network/sge/reward/campaign/{house_uid}/{uid}";
```

Comment:
since house id is not required this change also is not required

Reply:
removed, temporarily